mods/ra/weapons/missiles.yaml

```diff
@@ -401,10 +269,10 @@ TorpTube:
 		ValidImpactTypes: Water
 
 SubMissile:
+	Inherits: ^AntiGroundMissile
```

Comment:
Minor nit/suggestion: it might be worth `-Projectile`ing here too.  This is another case where you're switching a Missile default to a Bullet.